deployment/terraform/sqs.tf

```diff
@@ -0,0 +1,50 @@
+# sqs.tf
+
+resource "aws_sqs_queue" "kf_lambda_sqs" {
+  name                       = "lambda-${var.application}-${var.environment}-sqs"
+  delay_seconds              = 0
+  max_message_size           = 262144
+  message_retention_seconds  = 86400
+  receive_wait_time_seconds  = 10
+  visibility_timeout_seconds = 60
```

Comment:
These were hard-coded in the original configuration, so I kept it the same here, but this can also be parameterized.

Reply:
It is probably fine.